batavia/modules/time.js

```diff
@@ -239,21 +248,28 @@ time.localtime = function(seconds) {
         throw new exceptions.TypeError.$pyclass('localtime() takes at most 1 argument (' + arguments.length + ' given)')
```

Comment:
At column 5: [(brace-style) Opening curly brace does not appear on the same line as controlling statement.](http://eslint.org/docs/rules/brace-style)